base/acme/sbin/pki-acme-run

```diff
@@ -0,0 +1,184 @@
+#!/bin/sh
+#
+# Copyright Red Hat, Inc.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+
+. /usr/share/pki/scripts/config
+
+echo "################################################################################"
+
+# import ca_signing.cert and ca_signing.key if available
+if [ -f /var/lib/tomcats/pki/conf/certs/ca_signing.crt -a \
```

Comment:
what does `-a` mean in this context? if it means AND why not use `&&` ?

From man pages:
````
NOTE: Binary -a and -o are inherently ambiguous.  Use 'test EXPR1 && test  EXPR2'  or
       'test EXPR1 || test EXPR2' instead.
````

Reply:
Let me fix this.

Reply:
Fixed.

Reply:
Hmm.. I'm getting this now:
```
/usr/share/pki/acme/sbin/pki-acme-run: line 13: [: missing `]'
```

Reply:
I need to use double brackets.

Reply:
Nevermind. It should work now.